prov/mrail/src/mrail_init.c

```diff
@@ -207,6 +207,57 @@ static void mrail_adjust_info(struct fi_info *info, const struct fi_info *hints)
 	}
 }
 
+static struct fi_info *mrail_get_prefix_info(struct fi_info *core_info)
+{
+	struct fi_info *fi;
+	uint32_t num_rails;
+
+	for (fi = core_info, num_rails = 0; fi; fi = fi->next, ++num_rails);
```

Comment:
nit: please move ; to next line to clearly indicate that this is an empty for loop, rather than a typo.

Reply:
Done.